requirements/requirements.dev.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.8
+# This file is autogenerated by pip-compile with Python 3.12
```

Comment:
Does it need to compile with 3.11? It will use your local python to compile the way the scripts are written (we may want to fix that to use a locked python version)

Reply:
it does compiled with 3.11.9 and also tested in test environment
i have verified my local environment at 3.11.9